dtfs-central-api/api-tests/v1/portal-facility-amendments/submit-amendment-patch.api-test.ts

```diff
@@ -53,7 +62,7 @@ describe('PATCH /v1/portal/facilities/:facilityId/amendments/:amendmentId/submit
       .to('/v1/portal/gef/facilities');
 
     facilityId = createFacilityResponse.body._id;
-    referenceNumber = `${facilityId}-23`;
+    referenceNumber = `${new ObjectId().toString()}-23`;
```

Comment:
If this is the amendment reference number then it constructed based on UKEF Facility ID - Counter and not the Facility MongoDB ID i.e. `0040012345-23`